<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>1105</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="tag">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>-2</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Type/Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tag_co">
    <property name="geometry">
     <rect>
      <x>81</x>
      <y>10</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="res_num">
    <property name="geometry">
     <rect>
      <x>191</x>
      <y>9</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Result No.</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="res_num_co">
    <property name="geometry">
     <rect>
      <x>257</x>
      <y>9</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="open">
    <property name="geometry">
     <rect>
      <x>316</x>
      <y>11</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Open?</string>
    </property>
   </widget>
   <widget class="QComboBox" name="open_co">
    <property name="geometry">
     <rect>
      <x>355</x>
      <y>6</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>open</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>not matter</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="location">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>MY LOCATION</string>
    </property>
   </widget>
   <widget class="QLabel" name="longitude">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Longitude</string>
    </property>
   </widget>
   <widget class="QLabel" name="latitude">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>50</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Latitude</string>
    </property>
   </widget>
   <widget class="QLabel" name="distance">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>70</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Distance</string>
    </property>
   </widget>
   <widget class="QPushButton" name="enter">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>search</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>40</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>clear</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lo_co">
    <property name="geometry">
     <rect>
      <x>526</x>
      <y>30</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>-71.42</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="la_co">
    <property name="geometry">
     <rect>
      <x>526</x>
      <y>50</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>41.82</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dis_co">
    <property name="geometry">
     <rect>
      <x>526</x>
      <y>70</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QCustomPlot" name="map" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>737</width>
      <height>580</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>629</x>
      <y>52</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>( 41.813, 41.835 )</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>629</x>
      <y>30</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>(-71.425,-71.391)</string>
    </property>
   </widget>
   <widget class="QComboBox" name="sort_co">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>40</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>distance</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>review_count</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>stars</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="sort">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>43</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Sort</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionWelcome_to_Providence">
   <property name="text">
    <string>Welcome to Providence</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
